app/assets/javascripts/discourse/models/composer.js.es6

```diff
@@ -195,6 +195,11 @@ const Composer = RestModel.extend({
     return canEditTopicFeaturedLink ? !this.siteSettings.topic_featured_link_onebox : true;
   },
 
+  @computed("topic")
+  draftTitle(topic) {
+    return topic ? " <i class='fa fa-reply'></i> " + emojiUnescape(escapeExpression(topic.get('title'))) : I18n.t('composer.saved_draft');
```

Comment:
Aww no :heart:  for [Template Literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals)? :stuck_out_tongue: